flow/activities/flowable.go

```diff
@@ -1063,3 +1067,69 @@ func (a *FlowableActivity) RemoveFlowEntryFromCatalog(ctx context.Context, flowN
 
 	return nil
 }
+
+func (a *FlowableActivity) HandleCancelWorkFlow(ctx context.Context, workflowID string) error {
+	errChan := make(chan error, 1)
```

Comment:
Trying to get rid of this pattern, it's really a misuse of channels

Instead you should prefer a `chan struct{}` which closes after setting an `err` variable